### ISSUE TITLE
Error parsing regex

### ISSUE THREAD

**Issue by durka** [BurntSushi/ripgrep#166]
I keep seeing "Error parsing regex", even if I pass -F. But the program continues anyway.

Example:

```sh
$ rg -Ft rust Handle::new crates
Error parsing regex near '\\])$)' at character offset 28: Unicode features are not allowed when the Unicode (u) flag is not set.
crates/back/scribe/src/lib.rs
48:                let mut max_file = Handle::new();
49:                let mut max_pattern = Handle::new();
```

**Comment by BurntSushi** [comment]
I wonder if it's coming from parsing a `gitignore` file. Are you using the latest version of `ripgrep`? (`0.2.2`.) If so, is there anyway for you to narrow down which `gitignore` file is causing this and share it? Does this error appear for all patterns that you try to search?

**Comment by durka** [comment]
I have 0.2.1. There's a lot of gitignore files in this directory, is there any logging I can turn on to see which it is?

**Comment by BurntSushi** [comment]
You can try `--debug`.

- Andrew

**Comment by durka** [comment]
Error disappears with version 0.2.2.

**Comment by BurntSushi** [comment]
Ah great, I fixed a few bugs around these errors in `0.2.2`.
